src/serviceWorker.ts

```diff
@@ -14,6 +14,11 @@ chrome.runtime.onInstalled.addListener(async (details) => {
   }
 });
 
+// Subscribe to an event so that Chrome runs our service worker on startup
+chrome.runtime.onStartup.addListener(() => {
```

Comment:
thanks, let me try this build!

Also discovered that we need to get rid of `setInterval` here, as service workers don't support them really - https://developer.chrome.com/docs/extensions/develop/migrate/to-service-workers#convert-timers

Reply:
moved to https://github.com/artie-shevchenko/pr-review-your-turn/issues/15